### ISSUE TITLE
Missing check for Reversed Text on Kamikaze microgame

### ISSUE THREAD

**Issue by EgocentricPoliceDick** [gemidyne/microtf2#61]
The "Kamikaze is here!" text for the kamikaze player isn't affected by the Reversed Text special round modifier.

I've resolved this within my private fork within `Minigames/Minigame10.sp`:
```sp
	if (SpecialRoundID == 19)
	{
		ShowAnnotation(client, 3.0, "!ereh si ezakimaK");
	}
	else
	{
		ShowAnnotation(client, 3.0, "Kamikaze is here!");
	}
```

Just a small oversight, that's all.

**Comment by stevu236** [comment]
I just tried testing this on my test server but the text does reverse correctly. In Stocks.sp, there is an if statement in ShowAnnotationWithBitfield to handle the reverse text. Could you check your private fork for this?

**Comment by EgocentricPoliceDick** [comment]
I see the code you're talking about. Lines 132-160 if I'm not wrong?
We haven't touched stocks.sp as far as I know

**Comment by stevu236** [comment]
Yeah, this does the trick: https://github.com/gemidyne/microtf2/blob/master/src/scripting/Stocks.sp#L145

It should be in your fork if you haven't changed it

**Comment by EgocentricPoliceDick** [comment]
Ah, found it. Turns out we did have some changes (which somehow didn't include the code within the check for special 19...), including something that fixes #66 (we think) - I'll drop a comment in there with our code for it.

**Comment by stevu236** [comment]
OK, I'll close this issue for now.
